.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -103,10 +103,10 @@ schedules:
 # Global variables
 variables:
   buildConfiguration: Release
-  dotnetCoreSdkLatestVersion: 9.0.100
+  dotnetCoreSdkLatestVersion: 9.0.102
   # This is required until we're out of rc.
```

Comment:
Are these comments still relevant? (non-blocking)

Reply:
No, as we're not in RC 😅 But they _will_ be when we're in it again (october 😉) so I left them in with the latest changes 😄 Makes sense to clarify that though

Reply:
thanks!